frontend/src/components/common/Resource/ResourceTable.tsx

```diff
@@ -370,6 +377,41 @@ function ResourceTableContent<RowItem>(props: ResourceTableProps<RowItem>) {
     tableSettings,
   ]);
 
+  const defaultActions: RowAction[] = [
+    {
+      id: DefaultHeaderAction.EDIT,
+      action: ({ item, closeMenu }) => (
+        <EditButton item={item} menuAction afterConfirm={closeMenu} />
+      ),
+    },
+    {
+      id: DefaultHeaderAction.DELETE,
+      action: ({ item, closeMenu }) => (
+        <DeleteButton item={item} menuAction afterConfirm={closeMenu} />
+      ),
+    },
```

Comment:
For read-only roles, where users cannot delete nor edit resources, this will render the menu empty AFAIU.
One quick way to avoid this is to add a `VIEW` item. So at least we have an option in the menu. It's also something I have seen as an option sometimes, maybe it even improves a11y.

Reply:
I added VIEW.